packages/analytics/src/Analytics.js

```diff
@@ -1,35 +1,39 @@
-import React, { createContext, useContext, useRef, useEffect } from 'react';
-import { AvAnalytics } from '@availity/analytics-core';
+import React, { createContext, useContext, useEffect, useMemo } from 'react';
 import PropTypes from 'prop-types';
+import { AvAnalytics } from '@availity/analytics-core';
 
 export const AnalyticsContext = createContext();
 
+export const useAnalytics = () => {
+  const ctx = useContext(AnalyticsContext);
+  if (!ctx) throw new Error('useAnalytics must be used inside of the Analytics component');
```

Comment:
small nitpick but can we change to `useAnalytics must be used inside of the Analytics context` for clarity?